tfx/examples/imdb/imdb_utils_native_keras.py

```diff
@@ -0,0 +1,204 @@
+# Lint as: python2, python3
+# Copyright 2019 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Python source file include IMDB pipeline functions and necessary utils.
+
+The utilities in this file are used to build a model with native Keras.
+This module file will be used in Transform and generic Trainer."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from typing import List, Text
+
+import tensorflow as tf
+from tensorflow import keras
+import tensorflow_transform as tft
+
+from tfx.components.trainer.executor import TrainerFnArgs
+
+# There are 50,000 entries in the imdb dataset. ExampleGen splits the dataset
```

Comment:
If we use a smaller dataset, then will need to update this comment.

Reply:
added